web/docs/understand/data-model/schemas.md

```diff
@@ -0,0 +1,49 @@
+---
+sidebar_position: 1
+---
+
+# Schemas
+
+VAST defines a **schema** as a *named* record type. Since all types can have
+a name, we define a schema as a type a that (1) is a record, and (2) has a name.
+From a data perspective, the schema specifies the column structure of a
+batch of records.
+
+Ideally, each data source defines its own semantically rich schema to retain
+most of the domain-specific information of the data. This is desirable because
+accurately modeled data is more productive to work with because it's less
+error-prone to misinterpret and requires fewer context switches to infer missing
+gaps. [VAST's type system](type-system) is well-suited for deep domain modeling:
+it can express structure with lists and records, add meta data to any types via
+tags, and also support aliasing for building libraries of composable types.
+
+In practice, many tools often "dumb down" their rich internal representation
+into a generic piece of JSON, CSV, or text. This puts the burden of gaining
+actionable insights onto the analyst downstream: either they work with a minimal
```

Comment:
```suggestion
actionable insights onto the user downstream: either they work with a minimal
```

Can be any user, not just analysts. As this documentation is written independently from the user personas, it should just talk about users generally.